### PR TITLE
Component: Previous Trips/Hashtag Card PR

### PR DESCRIPTION
**Pull request by TylerEikenberg**
Issue #60 . This component renders different versions with the prop ```type``` of ```hashtags``` and ```trips```. Pass in an array of trips or hashtags using the prop ```data```. If the trips data array length is greater than 10 it is automatically trimmed down to only use the last 10 items.

### CODE REVIEW

**Comment by TylerEikenberg** [comment]
@ZakkMan I took your suggestions and significantly revamped this component. Now theres much less code, all the text content (besides the data) is passed in as children.

File: client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./PreviousTripHashtagCard.css";
+import "../../App.css";
+
+// Function based React Component
+const PreviousTripHashtagCard = props => {
+  console.log(props);
```
Comment (ZakkMan) [inline client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.js]: Remove and `console.log`s!

File: client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  // Default Class to apply to Component
+  const classList = `prev-hash-card-container`;
```
Comment (ZakkMan) [inline client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.js]: If you're not going to change or update this class, then just add it as a string to the `className` property?

File: client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./PreviousTripHashtagCard.css";
+import "../../App.css";
+
+// Function based React Component
+const PreviousTripHashtagCard = props => {
```
Comment (ZakkMan) [inline client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.js]: Can you think of a more general, semantic name for this?

File: client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    data = props.data.slice(dataLength - 10, dataLength);
+  } else {
+    data = props.data;
+  }
+
+  return (
+    <div className={classList}>
+      <div className="trips-content-container">
+        {props.children}
+        {props.type === "trips slider" ? <div className="trip-slider-arrow" /> : null}
```
Comment (ZakkMan) [inline client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.js]: Maybe slider could be a child? Or we could have a `SliderCard` component? This feels like too many things in a single component

File: client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.stories.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  .add("Previous Trips, No Data", () => (
+    <PreviousTripHashtagCard data={[]} type="trips">
+      <h1>Previous Trips</h1>
+      <h2>None with CoTripper yet!</h2>
+      <a href="#">Book a Trip Now!</a>
+    </PreviousTripHashtagCard>
+  ))
+  .add("Previous Trips", () => (
+    <PreviousTripHashtagCard
+      data={["New York", "Mexico", "Wakanda", "Hawaii", "Japan", "Canada", "Italy"]}
```
Comment (ZakkMan) [inline client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.stories.js]: One thing to note is that these hashtags might be links. So the data that would come through here as a prop might look more like:

```js
data={[{link: '/new-york', title: 'New York'},{link: '/mexico', title: 'Mexico'},{link: '/wakanda', title: 'Wakanda'},]}
```

Comment (ZakkMan) [reply client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.stories.js]: You can maybe find a way to handle both, or have this list be a `ul` that the user passes in as a child (like the text)

**Pull request merged**